config/ember-try.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+
+const getChannelURL = require('ember-source-channel-url');
+
+module.exports = function() {
+  return Promise.all([
+    getChannelURL('release'),
+    getChannelURL('beta'),
+    getChannelURL('canary')
+  ]).then((urls) => {
+    return {
+      useYarn: true,
+      scenarios: [
+        {
+          name: 'ember-lts-2.18',
```

Comment:
Could be replaced by `ember-lts-3.8`?

Reply:
Is it already an LTS? Also see the other comment on supported Ember versions, then I can update this if required!

Reply:
According to this doc https://emberjs.com/releases/, `3.8` is already LTS. Digging a bit, the blueprint update has been merged only 5 days ago 😬. I think this scenario should be added. Sorry for the manual update I'm asking you to do.

IMHO, there is no point in supporting ember versions not supported by ember anymore. (Even though they most probably work.) So we should drop 2.18 support. But @ef4 may disagree.

Reply:
There is a lot of value in keeping support for older Ember versions. For one thing, it makes it easier for people to do upgrades, because they can upgrade addons like this one first, and then upgrade Ember itself.

It's fine to drop older Ember versions if they're hard to support, but if the addon already works under that version anyway, there's no reason not to support and test it.